cli/src/opts/forge.rs

```diff
@@ -242,12 +246,10 @@ const VERSION_SEPARATOR: char = '@';
 impl FromStr for Dependency {
     type Err = eyre::Error;
     fn from_str(dependency: &str) -> Result<Self, Self::Err> {
-        // TODO: Is there a better way to normalize these paths to having a
-        // `https://github.com/` prefix?
-        let path = if dependency.starts_with("https://") {
-            dependency.to_string()
-        } else if dependency.starts_with(GITHUB) {
-            format!("https://{}", dependency)
+        let url_with_version = if let Some(captures) = GH_REPO_PREFIX_REGEX.captures(dependency) {
+            let brand = captures.get(5).unwrap().as_str();
```

Comment:
Is it safe to unwrap here always?

Reply:
I believe so, there are always `n` matches in the regex string, and the matches I am unwrapping here are not option so it wouldn't get to this line if this was `None` because the `match` would fail.